release_util/management/commands/generate_history.py

```diff
@@ -116,8 +116,11 @@ def handle(self, *args, **options):
                             """.format(
                                 table=table,
                                 historical_table=historical_table,
-                                insert_columns=','.join(columns),
-                                select_columns=','.join(['t.{}'.format(c) for c in columns]),
+                                # this cmd fails for tables containing reserved keywords in column.
+                                # https://dev.mysql.com/doc/refman/5.5/en/glossary.html
```

Comment:
Nit: this is a link to the general glossary, not very useful in this case.  Did you mean https://dev.mysql.com/doc/refman/5.7/en/keywords.html ?

Reply:
i meant to give docs link for using backticks here. Didn't have a hash url for backtick here so i mentioned this link.